app/src/main/java/org/mozilla/fenix/settings/account/AccountSettingsFragment.kt

```diff
@@ -250,7 +250,10 @@ class AccountSettingsFragment : PreferenceFragmentCompat() {
     private fun updateSyncEngineState(engine: SyncEngine, newValue: Boolean) {
         SyncEnginesStorage(requireContext()).setStatus(engine, newValue)
         viewLifecycleOwner.lifecycleScope.launch {
-            requireContext().components.backgroundServices.accountManager.syncNow(SyncReason.EngineChange)
+            requireContext().components.backgroundServices.accountManager.syncNow(
+                reason = SyncReason.EngineChange,
+                debounce = true,
```

Comment:
Same here as above - the user has explicitly changed the sync engines, so we shouldn't debounce here.

Reply:
I'll remove.  Thanks,